code/FormSchemaController.php

```diff
@@ -67,7 +67,7 @@ public function setFormSchema(FormSchema $schema): static
     }
 
     /**
-     * Gets a JSON schema representing the current edit form.
+     * Gets a JSON schema representing a form.
```

Comment:
Unrelated change but the form isn't necessarily (or even usually) the "current edit form" (implying main record edit form which isn't in a modal).